### PR TITLE
Japanese translate

### PR DESCRIPTION
**Pull request by ren-ozeki**
Related issue: Nothing
Related PR : #20939 

**Description**

I started to translate the documents of three.js into Japanese.
For now, I translate manual/.

### CODE REVIEW

File: docs/list.json [hunk 1]
```diff
@@ -1,1048 +1,936 @@
 {
-
-	"en": {
```
Comment (Mugen87) [inline docs/list.json:3]: The formatting of the existing sections should not be changed.

Comment (Mugen87) [reply docs/list.json:3]: This doesn't look correct yet. The diff is way to large...

Comment (na2shell) [reply docs/list.json:3]: I'm sorry. 
I thought it was fixed and pushed it, but I guess it wasn't.
I'll check locally again.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**